charts/service/values.yaml

```diff
@@ -102,7 +102,7 @@ datadog: {}
   # service: service_name
   # version: version
 
-preReleaseJob:
+preReleaseJobs:
```

Comment:
I think this should stay as `Job`, our template uses `preReleaseJob` not `Jobs` (`.Values.preReleaseJob.enabled`).

Reply:
Yep, this is accurate, no other repos have the string `preReleaseJobs` yet 23 have the string `preReleaseJob`. Somehow this wasn't working for me earlier. Will fix.

Reply:
Done